### ISSUE TITLE
Got error code 9999 when requesting '/spot/assetList'

### ISSUE THREAD

**Issue by olchaeng** [bithumb-pro/bithumb.pro-official-api-docs#17]
HI. I get an error when requesting '/spot/assetList'.

Public requests are fine.

request parameters : 
{'apiKey': 'not_realf_jlk21jl213jk1212132jkl3jk2l11', 'version': 'V1.0.0', 'msgNo': '', 'timestamp': 1571202541657, 'assetType': 'spot', 'signature': '2905951606597825554a137a85f4e2ce620f5ecd1e4f3aa2e3fdf85f0d1b4895'}

response:
'{"data":null,"code":"9999","msg":"system error","timestamp":1571202541845}'

What is the problem you expect?

**Comment by olchaeng** [comment]
sorry. It's my fault. I loaded params on the url instead of body.

**Comment by olchaeng** [comment]
Could you check this error?

body data is '{"apiKey": "not_real_4e73ff7d51f46fbf64e1af4c", "version": "V1.0.0", "msgNo": "", "timestamp": 1571204585985, "assetType": "spot", "signature": "b2cb3bc613f10bc60f1ab030905a28d3425974d01c136be6bb160686107bfcd6"}'

I got same error above '{"data":null,"code":"9999","msg":"system error","timestamp":1571204586197}'

I want to get specific error reason.

please help me.

**Comment by olchaeng** [comment]
I have some small work. 

This is body data.
b'{"apiKey": "not_real_055de8af17e54d0a74c734e11726c809cabadsfjksdfjlasdjkld", "msgNo": 1571207377275, "timestamp": 1571207377275, "assetType": "spot", "bizCode": "assetList", "signature": "not_real_3440f2982a6a6315e71faf9ebab0338516336200fcaa7ea36ddc308f"}'

and then I got a message '{"data":null,"code":"9000","msg":"missing parameter","timestamp":1571207380102}'

I think data which I loaded is fulfilled API docs. But It is not working.
I have no idea for that.
